docs/dapp/sapphire/precompiles.md

```diff
@@ -220,10 +231,16 @@ The `method`, `context_or_digest` and `message` parameters have the same meaning
 
 The algorithm-specific base cost below, with an additional 8 gas per 32 bytes of `context` and `message` for the `Ed25519Oasis`, `Ed25519Pure` and `Secp256k1Oasis` algorithms.
 
-* `0` (`Ed25519Oasis`), `1` (`Ed25519Pure`), `2` (`Ed25519PrehashedSha512`)
-  * 110,000 gas
-* `3` (`Secp256k1Oasis`), `4` (`Secp256k1PrehashedKeccak256`), `5` (`Secp256k1PrehashedSha256`)
-  * 210,000 gas
+* Ed25519: 2,000 gas
+  * `0` (`Ed25519Oasis`)
+  * `1` (`Ed25519Pure`)
+  * `2` (`Ed25519PrehashedSha512`)
+* Secp256k1: 3,000 gas
+  * `3` (`Secp256k1Oasis`)
+  * `4` (`Secp256k1PrehashedKeccak256`)
+  * `5` (`Secp256k1PrehashedSha256`)
+* Secp256r1: 7,900 gas
```

Comment:
Did you mean?

```suggestion
* Secp256r1: 9,000 gas
```

Reply:
Nope.

Verify: 7900 gas: https://github.com/oasisprotocol/oasis-sdk/blob/eb97a8162f84ae81d11d805e6dceeeb016841c27/runtime-sdk/modules/evm/src/precompile/confidential.rs#L71

Sign: 9000 gas: https://github.com/oasisprotocol/oasis-sdk/blob/eb97a8162f84ae81d11d805e6dceeeb016841c27/runtime-sdk/modules/evm/src/precompile/confidential.rs#L58